package.json

```diff
@@ -66,6 +66,7 @@
     "resize-observer-polyfill": "~1.5.1",
     "router5": "~7.0.2",
     "router5-plugin-browser": "~7.0.2",
+    "semver": "^7.3.2",
```

Comment:
Could you pls. tweak this to use the `~` qualifier, like the others?  That means we opt-in to minor version updates, which I think has been reasonable so far.